### PR TITLE
Feature/put request

### PR DESCRIPTION
**Pull request by piperjarrett**
What does this pr do? 
Creates a put request that successfully can change the rating based on what the user inputs. Also adds prototypes to each component

### CODE REVIEW

**Comment by piperjarrett** [comment]
I just want to make sure I am doing the PropTypes correct

File: src/Components/App/App.js [hunk 1]
```diff
@@ -72,4 +79,9 @@ function App() {
   );
 }
 
+App.propTypes = {
+  error: PropTypes.string,
+  cocktail: PropTypes.array,
```
Comment (Universal-Patois) [inline src/Components/App/App.js:84]: I think you could even get a little more specific if you wanted by using this `// An array of a certain type
  optionalArrayOf: PropTypes.arrayOf(PropTypes.number)`

**Review by Universal-Patois — COMMENTED** [review]
Looks good! I'm not super familiar with propTypes yet, but it looks like you have everything covered. My only suggestion would be you could define what your arrays will contain and could even use .shape to define your objects. Not sure if its necessary though!

**Comment by willhobson85** [comment]
Hi Piper! Sorry, I'm just seeing this. The propTypes look good to me. It makes clear what type of data needs to be used. You could also use .required at the end (like ingredients) for making sure the user doesn't search for nothing.

**Pull request merged**